rc4me/util_test.py

```diff
@@ -18,21 +13,35 @@ def check_repo_files_in_home(repo: Path):
 
 def test_get():
     runner = CliRunner()
-    result = runner.invoke(cli, ["get", "jeffmm/vimrc"])
+    result = runner.invoke(cli, ["get", "mstefferson/rc-demo"])
     assert result.exit_code == 0
-    repo = Path("~/.rc4me/jeffmm_vimrc").expanduser()
+    repo = Path("~/.rc4me/mstefferson_rc-demo").expanduser()
     assert repo.exists()
     assert repo.is_dir()
     check_repo_files_in_home(repo)
 
 
 def test_revert():
-    pass
+    runner = CliRunner()
+    result = runner.invoke(cli, ["get", "mstefferson/rc-demo"])
+    assert result.exit_code == 0
+    ms_repo = Path("~/.rc4me/mstefferson_rc-demo").expanduser()
+    check_repo_files_in_home(ms_repo)
```

Comment:
Since the files in `"~/.rc4me/mstefferson_rc-demo"` are a fixture, it would be easy to check the contents. This could be future work though.

Reply:
Maybe too much, but it would be good to be sure. Especially for the `init`